apps/src/templates/instructions/topInstructionsHelpers.js

```diff
@@ -0,0 +1,34 @@
+import $ from 'jquery';
```

Comment:
A few thoughts on this:
1) It would be nice if these methods could be easily recognizable as server calls when they're used.  One possibility night be to always reference them as e.g. `serverApi.getRubric()`.
2) Along the same lines, perhaps we can name this file something different?  ("Helpers" seems a bit vague and I'm worried we'll start putting other stuff in here that reduces clarity.)
3) I'm wondering if we want to abstract a bit more here and have these methods return a promise that returns something a bit more typed (e.g. returning an array of feedback vs. just the raw request).  I think this is a more natural abstraction and also has the benefit of improving unit testing for consuming code (because you can mock the server call more completely).  This probably needs some more thinking and can be a separate improvement.

Reply:
I've renamed `topInstructionsHelpers` to `topInstructionsDataApi` and imported the functions under that name so they're now referenced like `topInstructionsDataApi.getRubric()`. Let me know what you think, happy to rename again